src/test/java/org/prebid/server/bidder/intertech/IntertechBidderTest.java

```diff
@@ -0,0 +1,369 @@
+        return impCustomizer.apply(Imp.builder()
+                .banner(Banner.builder().w(100).h(100).build())
+                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpIntertech.of(123456, 7)))))
+                .build();
+    }
+
+    private static BidResponse givenBidResponse(Function<Bid.BidBuilder, Bid.BidBuilder> bidCustomizer) {
+        return BidResponse.builder()
+                .cur("USD")
+                .seatbid(singletonList(SeatBid.builder()
+                        .bid(singletonList(bidCustomizer.apply(Bid.builder()).build()))
+                        .build()))
+                .build();
+    }
+
+    private static BidderCall<BidRequest> givenBidderCall(BidRequest bidRequest, String body) {
+        return BidderCall.succeededHttp(
+                HttpRequest.<BidRequest>builder().payload(bidRequest).build(),
+                HttpResponse.of(200, null, body),
+                null);
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorIfImpExtCouldNotBeParsed() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                impBuilder -> impBuilder
+                        .ext(mapper.valueToTree(ExtPrebid.of(null, mapper.createArrayNode()))),
+                identity());
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = intertechBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).hasSize(1);
+        assertThat(result.getErrors().get(0).getMessage()).startsWith("imp #0: Cannot deserialize value");
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorWhenPageIdIsEmpty() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                impBuilder -> impBuilder
+                        .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpIntertech.of(null, 7)))),
+                identity());
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = intertechBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).hasSize(1)
+                .containsOnly(BidderError.badInput("imp #0: missing param page_id"));
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorWhenImpIdIsEmpty() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                impBuilder -> impBuilder
+                        .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpIntertech.of(123456, 0)))),
+                identity());
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = intertechBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).hasSize(1)
+                .containsOnly(BidderError.badInput("imp #0: missing param imp_id"));
```

Comment:
`.hasSize(1).containsOnly` can be simplified to `.containsExactly`. Also, please, check for similar occurences.